packages/accounts/src/light-account/accounts/base.ts

```diff
@@ -73,6 +81,7 @@ export type CreateLightAccountBaseParams<
   abi: Abi;
   signer: TSigner;
   accountAddress: Address;
+  accountType: TLightAccountType;
```

Comment:
No need. You can just do the type field of version (which is version def). I do that already so you can use the same type var.

Reply:
Ah thanks, that's good to know. Will change.